proxy/src/main/java/net/md_5/bungee/BungeeCord.java

```diff
@@ -770,16 +770,10 @@ public void removeConnection(UserConnection con)
         {
             return Collections.singleton( exactMatch );
         }
-
-        return Sets.newHashSet( Iterables.filter( getPlayers(), new Predicate<ProxiedPlayer>()
-        {
-
-            @Override
-            public boolean apply(ProxiedPlayer input)
-            {
-                return ( input == null ) ? false : input.getName().toLowerCase( Locale.ROOT ).startsWith( partialName.toLowerCase( Locale.ROOT ) );
-            }
-        } ) );
+        String lowerPartialName = partialName.toLowerCase( Locale.ROOT );
+        return getPlayers().stream()
+                .filter( (input) -> ( input == null ) ? false : input.getName().toLowerCase( Locale.ROOT ).startsWith( lowerPartialName ) )
+                .collect( Collectors.toCollection( HashSet::new ) );
```

Comment:
Just use
.collect( Collectors.toSet() );

Reply:
Returns an undefined set

Reply:
It was a hashset before and i don't want do change the behavior